controllers/controllers.go

```diff
@@ -84,6 +111,46 @@ func (r *instanceReconciler) ensureInstanceIsUpToDate(instanceInfo *instance.Inf
 	return nc.Configure()
 }
 
+var OverrideLabel = "windowsmachineconfig.openshift.io/force-upgrade"
+
+// upgradeBlocked returns whether the upgrade should be blocked if an upgrade will break storage functionality
+func (r *instanceReconciler) upgradeBlocked(node *core.Node, upgradingToCSI bool) (bool, error) {
+	if !upgradingToCSI {
+		return false, nil
+	}
+	if _, present := node.GetLabels()[OverrideLabel]; present {
```

Comment:
the "other" MCO has a "forcefile" that folks use to get into all sorts of trouble (and then bring us their broken clusters). 

It might not be terrible to make this "verbosely apparent" that the customer has overridden this ( so when they come to us with a bug or a gather we know that they miiiight have forced it and broken it :smile:  )